src/main/java/com/ocelotslovebirds/birdhaus/ticker/FixedIntervalTicker.java

```diff
@@ -0,0 +1,25 @@
+package com.ocelotslovebirds.birdhaus.ticker;
+
+public class FixedIntervalTicker implements Ticker {
+    private final int interval;
+    private int counter;
+
+    /**
+     * @param interval How often the ticker should tick
+     */
+    public FixedIntervalTicker(int interval) {
+        this.interval = interval;
+        this.counter = 0;
+    }
+
+    public boolean tick() {
```

Comment:
We could potentially shorten this to `return ++counter % interval == 0;`

Reply:
@Zachary-Concordia is it safe to infinitely increment a variable? I've never thought about this before